cmd/describe/addon/cmd_test.go

```diff
@@ -0,0 +1,53 @@
+package addon
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/format"
+	"github.com/openshift/rosa/pkg/output"
+	"github.com/spf13/cobra"
+)
+
+func TestDescribeAddon(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "rosa describe addon")
+}
+
+var _ = Describe("describe addon", func() {
+	Context("describe addon command", func() {
+		It("returns command", func() {
+			cmd := NewDescribeAddonCommand()
+			Expect(cmd).NotTo(BeNil())
+		})
+	})
+
+	Context("execute describe addon", func() {
+		// Full diff for long string to help debugging
+		format.TruncatedDiff = false
+
+		var cmd *cobra.Command
+
+		BeforeEach(func() {
+			output.SetOutput("")
+			cmd = NewDescribeAddonCommand()
+		})
+
+		AfterEach(func() {
+			output.SetOutput("")
+		})
+
+		It("Fails with no addon arg", func() {
+			err := cmd.Args(cmd, []string{})
+			Expect(err).ToNot(BeNil())
+			Expect(err.Error()).To(Equal("Expected exactly " +
+				"one command line argument containing the identifier of the add-on"))
+		})
+
+		It("Succeeds with addon arg", func() {
+			err := cmd.Args(cmd, []string{"name"})
+			Expect(err).To(BeNil())
+		})
```

Comment:
Might be best to include some more tests, maybe including mocking the API for a full end-to-end test of this command. If you want to do full testing of the command, here's an example in [a recent MR](https://github.com/openshift/rosa/pull/2054/files#diff-6d7a1f750109bd069d70a060cbfd0b0e52ec11aecd36e204f3cdb6fc9dd07a62)